impl/src/main/java/com/sun/faces/util/Util.java

```diff
@@ -505,6 +509,32 @@ public static void notNullAttribute(String attributeName, Object attribute) {
         }
     }
 
+    public static void notNullArgs( Object arg1 , Object arg2 ) {
+        if ( arg1 == null || arg2 == null )
+            throw new NullPointerException();
+    }
+
+    public static void notNullArgs( Object arg1 , Object arg2 , Object arg3 ) {
+        notNullArgs(arg1,arg2);
+        Objects.requireNonNull(arg3);
+    }
+
+    public static void notNullArgs( Object arg1 , Object arg2 , Object arg3 , Object arg4 ) {
+        notNullArgs(arg1,arg2,arg3);
+        Objects.requireNonNull(arg4);
+    }
+
+    public static void notNullArgs( Object arg1 , Object arg2 , Object arg3 , Object arg4 , Object arg5 ) {
+        notNullArgs(arg1,arg2,arg3,arg4);
+        Objects.requireNonNull(arg5);
+    }
+
+    public static void notNullArgs( Object... objets ) {
+        Objects.requireNonNull(objets);
+        for ( Object obj : objets)
+            Objects.requireNonNull(obj);
```

Comment:
`Arrays.stream(objects).forEach(Objects::requireNonNull)` :)

Reply:
I like a lot lambdas but I bet is cheaper to do a simple for ;)